azure/Guidelines.md

```diff
@@ -760,79 +760,67 @@ while the Microsoft guidelines use the name "Operation-Location".
 
 :white_check_mark: **DO** look for **both** HEADERS in client code, preferring the `Operation-Location` version.
 
-### Bring your own Storage
-When implementing your service, it is very common to store and retrieve data and files. When you encounter this scenario, avoid implementing your own storage strategy and instead use Azure Bring Your Own Storage (BYOS). BYOS provides significant benefits to service implementors, e.g. security, an aggressively optimized frontend, uptime, etc.
-While Azure Managed Storage may be easier to get started with, as your service evolves and matures, BYOS will provide the most flexibility and implementation choices. Further, when designing your APIs, be cognizant of expressing storage concepts and how clients will access your data. For example, if you are working with blobs, then you should not expose the concept of folders, nor do they have extensions.
+### Bring your own Storage (BYOS)
+Many services need to store and retrieve data files. For this scenario, the service should not implement its own 
+storage APIs and should instead leverage the existing Azure Storage service. When doing this, the customer 
+"owns" the storage account and just tells your service to use it. Colloquially, we call this <i>Bring Your Own Storage</i> as the customer is bringing their storage account to another service. BYOS provides significant benefits to service implementors: security, performance, uptime, etc. And, of course, most Azure customers are already familiar with the Azure Storage service. 
 
-:white_check_mark: **DO** use Azure Bring Your Own Storage.
+While Azure Managed Storage may be easier to get started with, as your service evolves and matures, BYOS provides the most flexibility and implementation choices. Further, when designing your APIs, be cognizant of expressing storage concepts and how clients will access your data. For example, if you are working with blobs, then you should not expose the concept of folders.
```

Comment:
Is “Azure Managed Storage” different from the “Azure Storage service”? If so, will that be obvious to readers?  Do we need to clarify that?